src/cms/models/languages/language_tree_node.py

```diff
@@ -37,45 +58,122 @@ class LanguageTreeNode(MPTTModel):
 
     @property
     def code(self):
+        """
+        Returns the code of this node's language
+
+        :return: The language code of this language node
+        :rtype: str
+        """
         return self.language.code
 
     @property
     def native_name(self):
+        """
+        Returns the native name of this node's language
+
+        :return: The native name of this language node
+        :rtype: str
+        """
         return self.language.native_name
 
     @property
     def english_name(self):
+        """
+        Returns the name of this node's language in English
+
+        :return: The English name of this language node
+        :rtype: str
+        """
         return self.language.english_name
 
-    # name of the language in the current backend language
     @property
     def translated_name(self):
+        """
+        Returns the name of this node's language in the current backend language
+
+        :return: The translated name of this language node
+        :rtype: str
+        """
         return self.language.translated_name
 
     @property
     def text_direction(self):
+        """
+        Returns the text direction (e.g. left-to-right) of this node's language
+
+        :return: The text direction name of this language node
+        :rtype: str
+        """
         return self.language.text_direction
 
     @property
     def depth(self):
-        """Provide level of inheritance
-
-        Returns:
-            Int : Number of ancestors
         """
+        Counts how many ancestors the node has. If the node is the root node, its depth is `0`.
 
+        :return: The depth of this language node
+        :rtype: str
+        """
         return len(self.get_ancestors())
 
+    # Explicitly define functions to show documentation of base model
+    @raise_if_unsaved
```

Comment:
Just something that looks weird to me: why does useless-super-delegation have to be disabled for pylint for some functions but not for others? The functions below should theoretically all trigger this pylint warning/error, or am I missing something here?

Reply:
Actually, I have no idea why. I think it's a bug in pylint :sweat_smile: 
When I ran pylint, only the three functions caused a warning and I don't see any reason why. But maybe I'm missing something.